aviary/subsystems/propulsion/test/test_turboprops.py

```diff
@@ -107,6 +110,9 @@ def test_case_1(self):
         truth_vals = [(112, 37.7, 195.8), (1120, 136.3, 644), (1742.5, 21.3, 839.7)]
         self.prepare_model(filename, test_points)
 
+        self.prob.setup(force_alloc_complex=True)
+        self.prob.set_val(Aircraft.Engine.SCALE_FACTOR, 1, units='unitless')
```

Comment:
This could probably get moved back into prepare_model, to reduce duplicate code.